yang/frr-isisd.yang

```diff
@@ -2078,6 +2125,45 @@ module frr-isisd {
         }
       }
 
+      container route_leaking {
```

Comment:
Repeating my previous review - this is not needed. We added the `level-from` leaf to the `redistribute` container, specifically to cover all of this. Why do you think this separate container is needed?

Reply:
I think we need such a container because in callbacks, for example, we parse the young model, and if we don't make a separate container for local redistribution, then we will deal with the same callback that is already provided in ISIS: 
`return nb_cli_apply_changes(vty,
				    "./route_leaking/%s[protocol='%s'][level='%s']",
				    ip, proto, level_from);`

Reply:
I don't understand the example you provided, but that's exactly the point - why do we need any additional callbacks for that?